lib/types/index.d.ts

```diff
@@ -255,70 +170,204 @@ declare module 'react-native-wifi-reborn' {
+            SSIDPrefix: string,
+            password: string | null,
+            isWEP: boolean,
+            joinOnce: boolean
+        ): Promise<void>;
+
+        //#endregion
+
+        //#region Android only
+
+        /**
+         * Returns a list of nearby WiFI networks.
+         */
+        loadWifiList(): Promise<Array<WifiEntry>>;
+
+        /**
+         * Similar to `loadWifiList` but it forcefully starts a new WiFi scan and only passes the results when the scan is done.
+         */
+        reScanAndLoadWifiList(): Promise<Array<WifiEntry>>;
+
+        /**
+         * Method to check if wifi is enabled.
+         */
+        isEnabled(): Promise<boolean>;
+
+        setEnabled(enabled: boolean): void;
+
+        /**
+         * Returns if the device is currently connected to a WiFi network.
+         */
+        connectionStatus(): Promise<boolean>;
+
+
+        disconnect(): Promise<boolean>;
+
+        /**
+         * Returns the BSSID (basic service set identifier) of the currently connected WiFi network.
+         */
+        getBSSID(): Promise<string>;
+
+        /**
+         * Returns the RSSI (received signal strength indicator) of the currently connected WiFi network.
+         */
+        getCurrentSignalStrength(): Promise<number>;
+
+        /**
+         * Returns the frequency of the currently connected WiFi network.
+         */
+        getFrequency(): Promise<number>;
+
+        /**
+         * Returns the IP of the currently connected WiFi network.
+         */
+        getIP(): Promise<string>;
+
+        /**
+         * This method will remove the wifi network configuration.
+         * If you are connected to that network, it will disconnect.
+         *
+         * @param SSID wifi SSID to remove configuration for
+         */
+        isRemoveWifiNetwork(SSID: string): Promise<boolean>;
+
+        /**
+         * @deprecated Use forceWifiUsageWithOptions.
+         */
+        forceWifiUsage(useWifi: boolean): Promise<void>;
+
+        /**
+         * Use this to route all app network requests to the wifi network (instead of the mobile connection).
+         * It is important to disable it again after using as even when the app disconnects from the wifi
+         * network it will keep on routing everything to wifi.
+         *
+         * Useful for commissioning IoT devices. If the wifi access point has no internet you can indicate so with
+         * the option `noInternet`.
+         *
+         * @param useWifi Force wifi usage on or off.
+         * @param options `noInternet` To indicate the access point has no internet. Usefull as some
+         * phone vendor customizations will switch back to mobile when the wifi access point has no internet.
+         */
+        forceWifiUsageWithOptions(
+            useWifi: boolean,
+            options: { noInternet: boolean }
+        ): Promise<void>;
+
+        //#endregion
+    }
 
-    //#endregion
+    var WifiManager: WifiManagerStatic;
```

Comment:
```suggestion
    const WifiManager: WifiManagerStatic;
```
does it make sense to replace var with const here?

Reply:
Yes, const makes sense.
In fact looking when looking at what other modules do, it seems usual that I should call the interface 'WifiManager' instead of 'WifiManagerStatic' to align with the native module name, and export via an const called '_default'.
e.g.:
```typescript
interface WifiManager { ... }
const _default : WifiManager;
export default _default;
```
This is pushed... let me know if its OK, or you prefer another way!